package.json

```diff
@@ -54,7 +54,7 @@
     "@netlify/edge-bundler": "^12.2.3",
     "@netlify/edge-functions": "^2.11.0",
     "@netlify/eslint-config-node": "^7.0.1",
-    "@netlify/functions": "^2.8.2",
+    "@netlify/functions": "^3.0.0",
```

Comment:
https://github.com/netlify/functions/releases/tag/v3.0.0

Breaking change is not affecting us, as runtime already is node@>=18: https://github.com/opennextjs/opennextjs-netlify/blob/f00b542100a39182ac2e32521705999267bcd55d/package.json#L12-L14